content/docs/1.8.0/advanced-resources/backing-image/backing-image.md

```diff
@@ -154,6 +159,32 @@ Since v1.3.0, users can download existing backing image files to the local via U
 #### Notice:
 - Users need to make sure the backing image existence when they use UI to create or restore a volume with a backing image specified.
 - Before downloading an existing backing image file to the local, users need to guarantee there is a ready file for it.
+- Downloading of V2 Data Engine backing images is currently not supported.
+
+## Create a V2 Data Engine Backing Image
+
+Starting v1.8.0, you can create a backing image that is supported by the V2 Data Engine by configuring `Data Engine` in the YAML (through the UI or a StorageClass).
+
+### Parameters During Creation
+
+All parameters are the same as that of the V1 Data Engine backing image, except for `Data Engine`.
+
+#### Backing Image Data Sources
+
+You can prepare a V2 Data Engine backing image using any of the supported data sources.
+- Download a backing image file (using a URL).
+- Upload a file from your local machine. This option is available to Longhorn UI users.
+- Export an existing in-cluster V1 Data Engine volume as a backing image.
+- Restore a backing image from the backupstore. For more information, see [Backing Image Backup](../backing-image-backup).
```

Comment:
Is restoring the backing image supported in v2? https://github.com/longhorn/longhorn/issues/9992

Reply:
Yes, it is actually supported naturally
The only part we haven't implemented yet is "**Backing up V2 BackingImage**"
By design, V2 and V1 BackingImage has the same format of the backup in the backupstore, they don't have any different.
Thus, one can restore the BackupBackingImage to V1 or V2 BackingImage in v1.8.0 already.

Reply:
Good. Thanks.

Reply:
We also tested it in the v2 backing image issue
https://github.com/longhorn/longhorn/issues/6341#issuecomment-2559348262